Sources/AnimalKingdomAPI/Generated/Query.swift

```diff
@@ -0,0 +1,9 @@
+import ApolloAPI
+
+public final class Query: Object {
```

Comment:
Is `Query` ever going to be useful/needed? Seems like an unintended consequence of pulling in all referenced types.

Reply:
I intentionally added it in this PR. It is 100% needed for the generated code to build and execute correctly. :)